bin/setup

```diff
@@ -5,26 +5,20 @@ set -e
 # Setup specific Bundler options if this is CI
 if [ "$CI" ]; then
   BUNDLER_ARGS="--without development staging production"
+  RACK_ENV="test"
 fi
 
-# Make sure we have Bundler installed
-gem install bundler --conservative
-
-# Set up Ruby dependencies via Bundler into .bundle folder
-rm -f .bundle/config
-
-bundle check --path .bundle > /dev/null 2>&1 ||
-  bundle install --path=.bundle $BUNDLER_ARGS
-
 # Set up configurable environment variables
 if [ ! -f .env ]; then
   cp .env.example .env
 fi
 
+# Build Docker image
+docker-compose build
```

Comment:
I think we should change way of thinking about setup process.

I prefer to use bin scripts inside containers and only inside. The idea of using docker-compose in dev env assume that developer may not have Ruby interpreter on her/him machine and ALL parts of dev environment should be concluded in container(s).